src/test/java/uk/gov/register/presentation/functional/FunctionalTest.java

```diff
@@ -41,6 +45,26 @@ public void shouldConsumeMessageFromKafkaAndShowAsLatest() throws Exception {
 
     }
 
+    @Test
+    public void appSupportsCORS() {
```

Comment:
is it possible to test this in a [Resource test](https://dropwizard.github.io/dropwizard/manual/testing.html#testing-resources) rather than a functional test? Functional tests are expensive and this behaviour isn't a property of the whole app, just of this one resource.